articles/connectivity/conn-faq-psn.md

```diff
@@ -141,9 +141,9 @@ Before additional IPv4 addresses can be allocated, you'll need to demonstrate yo
 
 PSN is a shared connection and all UKCloud PSN‑enabled customers use a shared bandwidth model.
 
-### Does each PSN environment have a maximum limit assigned to it?
+### Does each PSN connection have a maximum bandwidth limit assigned to it?
 
-We don't publish limits associated with PSN bandwidth as we capacity‑manage this link to ensure there is always enough bandwidth for our customers. Consumption of PSN bandwidth is based on the actual volume of data transferred (outbound) per month.
+We do not publish limits associated with PSN bandwidth as we capacity manage the link sto ensure there is always enough bandwidth for our customers. Consumption of PSN bandwidth is based on the actual volume of data transferred (outbound) per month.
```

Comment:
typo - the link sto ensure

Reply:
Fixed typo and reverted back to using contractions per KC style guidelines